dali/test/python/test_operator_numpy_reader.py

```diff
@@ -122,3 +216,62 @@ def check_array(filename, shape, typ, fortran_order=False):
 
     # delete temp files
     delete_numpy_file(filename)
+
+
+def check_array_static_slab(filename, shape, slab_anchor, slab_shape, typ, io_size=0, fortran_order=False):
+    # setup file
+    create_numpy_file(filename, shape, typ, fortran_order)
```

Comment:
Why not use `tempfile` as well and don't bother with `delete_numpy_file`?

Reply:
fixed